flow/envs/base.py

```diff
@@ -214,7 +214,14 @@ def __init__(self,
             # render a frame
             self.render(reset=True)
         elif self.sim_params.render in [True, False]:
-            pass  # default to sumo-gui (if True) or sumo (if False)
+            # default to sumo-gui (if True) or sumo (if False)
+            if (self.sim_params.render is True) and self.sim_params.save_render:
+                path = os.path.expanduser('~')+'/flow_rendering/'
```

Comment:
Why is there both a flow_rendering path and a flow_movies path?

Reply:
This was patterned after what already exists in `visualizer_rlllib.py`.

Reply:
I see. I think we should probably merge this into one folder instead of there being two. It seems like that would reduce complexity rather than raise it.

Reply:
Done. Sticking with `flow_rendering` for consistency with pyglet functionality.